composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name":"codeception/module-phalcon4",
+    "description":"Codeception module for Phalcon 4 framework",
+    "keywords":["codeception", "phalcon"],
+    "homepage":"https://codeception.com/",
+    "type":"library",
+    "license":"MIT",
+    "authors":[
+        {
+            "name":"Ruud Boon"
+        }
+    ],
+    "minimum-stability": "RC",
+
+    "require": {
+        "php": ">=7.2.0 <8.0",
+        "codeception/lib-innerbrowser": "dev-master | ^1.0",
+        "codeception/codeception": "4.0.x-dev | ^4.0",
```

Comment:
Please remove `4.0.x-dev` and all `dev-master` constraints, these were only used during development of Codeception 4.0.